python/lsst/afw/table/exposure.cc

```diff
@@ -25,23 +25,152 @@
 //#include <pybind11/stl.h>
 
 namespace py = pybind11;
+using namespace pybind11::literals;
+
+#include "lsst/afw/coord/Coord.h"
+#include "lsst/afw/detection/Psf.h"  // forward-declared by Exposure.h
+#include "lsst/afw/geom/Box.h"
+#include "lsst/afw/geom/Point.h"
+#include "lsst/afw/geom/polygon/Polygon.h"  // forward-declared by Exposure.h
+#include "lsst/afw/image/ApCorrMap.h"  // forward-declared by Exposure.h
+#include "lsst/afw/image/Calib.h"  // forward-declared by Exposure.h
+#include "lsst/afw/image/VisitInfo.h"  // forward-declared by Exposure.h
+#include "lsst/afw/image/Wcs.h"  // forward-declared by Exposure.h
+#include "lsst/afw/table/BaseRecord.h"
+#include "lsst/afw/table/BaseTable.h"
+#include "lsst/afw/table/Catalog.h"
+#include "lsst/afw/table/Exposure.h"
+#include "lsst/afw/table/pybind11/catalog.h"
+#include "lsst/afw/table/pybind11/sortedCatalog.h"
 
 namespace lsst {
 namespace afw {
 namespace table {
 
+typedef py::class_<CatalogT<ExposureRecord>,
+                   std::shared_ptr<CatalogT<ExposureRecord>>> PyBaseExposureCatalog;
+
+typedef py::class_<SortedCatalogT<ExposureRecord>,
+                   std::shared_ptr<SortedCatalogT<ExposureRecord>>,
+                   CatalogT<ExposureRecord>> PySortedBaseExposureCatalog;
+
+typedef py::class_<ExposureCatalogT<ExposureRecord>,
+                   std::shared_ptr<ExposureCatalogT<ExposureRecord>>,
+                   SortedCatalogT<ExposureRecord>> PyExposureCatalog;
```

Comment:
I typically prefer `using` but it doesn't really matter. I do suspect that at some point we will want to use `using` everywhere, and so far I think I have always used it in pybind11.